src/Microsoft.Health.Fhir.SqlServer/Features/Schema/Migrations/4.diff.sql

```diff
@@ -1,39 +1,31 @@
 /*************************************************************
-    Search Parameter Status Registry
+    Search Parameter Status Information
 **************************************************************/
 
-CREATE TYPE dbo.SearchParamStatusRegistryTableType_1 AS TABLE
-(
-    Uri varchar(128) COLLATE Latin1_General_100_CS_AS NOT NULL,
-    Status varchar(10) NOT NULL,
-    IsPartiallySupported bit NOT NULL
-)
+ALTER TABLE dbo.SearchParam
+ADD
+    Status varchar(10) NULL,
+    LastUpdated datetimeoffset(7) NULL,
+    IsPartiallySupported bit NULL
```

Comment:
This is different to the change made to 4.sql - in that file, the table definition is simply modified.

Reply:
These values are set to be `NULL` until they are initialized in `SqlServerFhirModel`.